es.json

```diff
@@ -236,6 +248,11 @@
   "fantoche": 2,
   "fariseo": 2,
   "fiesta de salchichas": 2,
+  "fiesta de pollas": 2,
+  "fiesta de putas": 2,
```

Comment:
We already have `puta` and `putas`. If `fiesta de putas` appears, it'll trigger two errors. I think we might be able to remove this, but I'll leave that decision up to the repo owner.